src/states/game.ts

```diff
@@ -163,7 +166,8 @@ export default class Title extends Phaser.State {
     private addExplosions(): void {
         this.explosions = this.game.add.group();
         for (let i = 0; i < 10; i++) {
-            let explosionAnimation = this.explosions.create(0, 0, 'kaboom', 0, false);
+            let explosionAnimation = this.explosions.create(0, 0,
```

Comment:
Maybe we could let the old explosion ... as it's done here we have a simple sprite, not an animation via spritesheet like before. I'm not even sure we can have an animation with the new Kenney assets.
This is open to discussion, just leaving my opinion here :smiley: